source/reference/method/MongoDBCollection-findOne.txt

```diff
@@ -27,154 +25,7 @@ Definition
-       - integer
-       - Maximum number of documents or index keys to scan when executing the
-         query.
-
-         .. deprecated:: 1.4
-         .. versionadded:: 1.2
-
-     * - maxTimeMS
-       - integer
-       - .. include:: /includes/extracts/common-option-maxTimeMS.rst
-
-     * - min
-       - array|object
-       - The inclusive lower bound for a specific index.
-
-         .. versionadded:: 1.2
-
-     * - modifiers
-       - array|object
-       - :manual:`Meta operators </reference/operator/query-modifier>` that
-         modify the output or behavior of a query. Use of these operators is
-         deprecated in favor of named options.
-
-     * - oplogReplay
-       - boolean
-       - Internal use for replica sets. To use ``oplogReplay``, you must include
-         the following condition in the filter:
-
-         .. code-block:: javascript
-
-            { ts: { $gte: <timestamp> } }
-
-         The :php:`MongoDB\BSON\Timestamp <class.mongodb-bson-timestamp>`
-         class reference describes how to represent MongoDB's BSON timestamp
-         type with PHP.
-
-         .. deprecated:: 1.7
-
-     * - projection
-       - array|object
-       - The :ref:`projection specification <projections>` to determine which
-         fields to include in the returned documents. See
-         :manual:`Project Fields to Return from Query </tutorial/project-fields-from-query-results>`
-         and :manual:`Projection Operators </reference/operator/projection>` in
-         the MongoDB manual.
-
-     * - readConcern
-       - :php:`MongoDB\Driver\ReadConcern <class.mongodb-driver-readconcern>`
-       - .. include:: /includes/extracts/collection-option-readConcern.rst
-
-         .. include:: /includes/extracts/common-option-readConcern-transaction.rst
-
-     * - readPreference
-       - :php:`MongoDB\Driver\ReadPreference <class.mongodb-driver-readpreference>`
-       - .. include:: /includes/extracts/collection-option-readPreference.rst
-
-     * - returnKey
-       - boolean
-       - If true, returns only the index keys in the resulting documents.
-
-         .. versionadded:: 1.2
-
-     * - session
-       - :php:`MongoDB\Driver\Session <class.mongodb-driver-session>`
-       - .. include:: /includes/extracts/common-option-session.rst
-
-         .. versionadded:: 1.3
-
-     * - showRecordId
-       - boolean
-       - Determines whether to return the record identifier for each document.
-         If true, adds a field ``$recordId`` to the returned documents.
-
-         .. versionadded:: 1.2
-
-     * - skip
-       - integer
-       - Number of documents to skip. Defaults to ``0``.
-
-     * - sort
-       - array|object
-       - The sort specification for the ordering of the results.
-
-     * - typeMap
-       - array
-       - .. include:: /includes/extracts/collection-option-typeMap.rst
-
-         This will be used for the returned result document.
+.. include:: /includes/reference/find-one-parameters.rst
```

Comment:
As with the `find` include, please confirm that this include exactly matches the content from the corresponding methods (e.g. no `versionadded` inconsistencies).

Reply:
same issue as above with the extracts